### ISSUE TITLE
unrecognised option '--output_path'

### ISSUE THREAD

**Issue by ynjiun** [colmap/colmap#1445]
sudo apt install colmap

then run colmap in python:

colmap mapper --database_path results/set2v1_short/colmap_dense/database.db --image_path results/set2v1_short/color_full --output_path results/set2v1_short/colmap_dense/sparse --Mapper.abs_pose_min_inlier_ratio 0.5 --Mapper.abs_pose_min_num_inliers 50 --Mapper.init_max_forward_motion 1 --Mapper.ba_local_num_images 15

when outputing results and got the following error message:

`ERROR: Failed to parse options: unrecognised option '--output_path'.`

Please advise how to fix this issue.

**Comment by ynjiun** [comment]
Can anyone confirm that --output_path option has been change to --export_path?

I can run the above colmap mapper command by replacing "--output_path" to "--export_path". But not sure if it is correct....

If it is true, why change? Thanks.

**Comment by ynjiun** [comment]
instead of "sudo apt install colmap",

I built colmap from source with cuda (see the installation guide [here](https://colmap.github.io/install.html))

Then this unrecognised option '--output_path' is gone!

This shows the inconsistency between "sudo apt install colmap" and built from source colmap. Don't have insight on why. Hopefully someone familar with colmap can help to clear out this confusion. Thanks.

**Comment by tangconghui2008** [comment]
I also use "sudo apt install colmap " for colmap install
You can try this:
  change  --output_path to --export_path
  change  --input_path   to  --inport_path

**Comment by kaixin-bai** [comment]
I've encountered an even stranger issue. I installed COLMAP using sudo apt-get install colmap. The first time I reconstructed a project using --output_path, there were no issues. However, starting from a certain point, using --output_path started throwing errors. Previously, a sparse point cloud would be generated in the sparse folder, but after the issue arose, it stopped generating the point cloud and instead created .bin files.
